Altis_Life.Altis/core/shops/fn_weaponShopMags.sqf

```diff
@@ -8,11 +8,20 @@
 */
 disableSerialization;
 
+private ["_muzzles","_eglArray","_magArray","_weapon"];
 if ((uiNamespace getVariable ["Weapon_Magazine",0]) isEqualTo 0) then {
-    private ["_weapon"];
     _weapon = lbData[38403,lbCurSel (38403)];
-    _weapon = FETCH_CONFIG2(getArray,"CfgWeapons",_weapon,"magazines");
-    uiNamespace setVariable ["Magazine_Array",_weapon];
+    _magArray = FETCH_CONFIG2(getArray,"CfgWeapons",_weapon,"magazines");
+    _muzzles = FETCH_CONFIG2(getArray,"CfgWeapons",_weapon,"muzzles");
+    if ("EGLM" in _muzzles) then {
```

Comment:
We can forEach/count over "EGLM" & "GL_3GL_F" to stray from duplicating this code